packages/paste-libraries/styling/src/css-function/css.ts

```diff
@@ -0,0 +1,224 @@
+// Very lightly TS Version of https://github.com/styled-system/styled-system/blob/master/packages/css/src/index.js
+// not typed correctly in the slightest
```

Comment:
Can you outline the specific lines you've changed from the src (logic wise) with comments to make future maintenance easier please
And can you also leave a note on which version of the css function you copied from Styled-System, preferably with a link to the commit/file in github